src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Main program for Duke.
+ */
+public class Duke {
+    /**
+     * Main program function.
+     * @param args input arguments, not used
+     */
+    public static void main(String[] args) {
+        Ui.printGreeting();
+        String userInput;
+        TaskList userTaskList = new TaskList();
+        Scanner scanner = new Scanner(System.in);
+        loop: while (true) {
+            userInput = scanner.nextLine();
+            try {
+                UserCommandType userCommandType = Parser.parseUserCommand(userInput);
+                switch (userCommandType) {
```

Comment:
Just a minor code styling issue, no indentations are for the different case clauses.